### PR TITLE
Make Travis passing again

### LINKED ISSUE #9571: Invalid CSRF token when cancelling order via Admin UI in v1.1.9
**Sylius version affected**: 1.1.9

```
Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\HttpException: "Invalid CSRF token." at /app/vendor/sylius/sylius/src/Sylius/Bundle/ResourceBundle/Controller/ResourceController.php line 541
```

```

Message: | Matched route "sylius_admin_order_cancel".
-- | --
Time: | 2018-07-10 10:40:44
Channel: | request
Context: | route:   sylius_admin_order_cancel     route_parameters:   {     "_controller": "sylius.controller.order:applyStateMachineTransitionAction",     "_sylius": {         "permission": true,         "state_machine": {             "graph": "sylius_order",             "transition": "cancel"         },         "redirect": "referer"     },     "id": "1",     "_route": "sylius_admin_order_cancel" }     request_uri:   https://www.host.com/admin/orders/1/cancel     method:   PUT | route: | sylius_admin_order_cancel | route_parameters: | {     "_controller": "sylius.controller.order:applyStateMachineTransitionAction",     "_sylius": {         "permission": true,         "state_machine": {             "graph": "sylius_order",             "transition": "cancel"         },         "redirect": "referer"     },     "id": "1",     "_route": "sylius_admin_order_cancel" } | request_uri: | https://www.host.com/admin/orders/1/cancel | method: | PUT
route: | sylius_admin_order_cancel
route_parameters: | {     "_controller": "sylius.controller.order:applyStateMachineTransitionAction",     "_sylius": {         "permission": true,         "state_machine": {             "graph": "sylius_order",             "transition": "cancel"         },         "redirect": "referer"     },     "id": "1",     "_route": "sylius_admin_order_cancel" }
request_uri: | https://www.host.com/admin/orders/1/cancel
method: | PUT
```

### PR DESCRIPTION
**Pull request by pamil**
Fixes #9571.

### CODE REVIEW

**Comment by alcaeus** [comment]
Can you release a bugfix release with this today? Unfortunately that's a bad regression that requires us to override some admin API routes. Just a heads up, for 1.1 and later, the `sylius_admin_api_product_review_accept` and `sylius_admin_api_product_review_reject` routes are also affected.

**Comment by Zales0123** [comment]
Unfortunately, there is still some problem with order cancellation scenario :/ ping @pamil

**Comment by pamil** [comment]
@alcaeus of course, as soon as the build is passing.

**Comment by pamil** [comment]
@Zales0123 should be fixed.

**Comment by lchrusciel** [comment]
Will it fix the #9574 problem as well?

**Comment by pamil** [comment]
@lchrusciel nope, I will merge #9574 directly to `1.0` branch.

File: src/Sylius/Bundle/AdminBundle/Menu/OrderShowMenuBuilder.php [hunk 1]
```diff
@@ -40,18 +41,20 @@
...
+     * @var CsrfTokenManagerInterface
      */
+    private $csrfTokenManager;
+
     public function __construct(
         FactoryInterface $factory,
         EventDispatcherInterface $eventDispatcher,
-        StateMachineFactoryInterface $stateMachineFactory
+        StateMachineFactoryInterface $stateMachineFactory,
+        CsrfTokenManagerInterface $csrfTokenManager
```
Comment (pamil) [inline src/Sylius/Bundle/AdminBundle/Menu/OrderShowMenuBuilder.php]: It's a security-related BC break, will be documented in UPGRADEs.md before the release.

**Pull request merged**